pkg/hintrunner/zero/zerohint_memcpy.go

```diff
@@ -28,3 +33,119 @@ func createVMExitScopeHinter(resolver hintReferenceResolver) (hinter.Hinter, err
 		},
 	}, nil
 }
+
+func createSearchSortedLowerHinter(resolver hintReferenceResolver) (hinter.Hinter, error) {
+	array_ptr, err := resolver.GetResOperander("array_ptr")
+	if err != nil {
+		return nil, err
+	}
+	elm_size, err := resolver.GetResOperander("elm_size")
+	if err != nil {
+		return nil, err
+	}
+	n_elms, err := resolver.GetResOperander("n_elms")
+	if err != nil {
+		return nil, err
+	}
+	key, err := resolver.GetResOperander("key")
+	if err != nil {
+		return nil, err
+	}
+	index, err := resolver.GetResOperander("index")
+	if err != nil {
+		return nil, err
+	}
+	return newSearchSortedLowerHint(array_ptr, elm_size, n_elms, key, index), nil
+}
+
+func newSearchSortedLowerHint(array_ptr, elm_size, n_elms, key, index hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "SearchSortedLower",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+
+			//> array_ptr = ids.array_ptr
+			array_ptr, err := hinter.ResolveAsAddress(vm, array_ptr)
```

Comment:
Can we stick to camelCase for variable names?
You'll have to fix this and the rest of the variables to use the same format